docs/changelog.md

```diff
@@ -10,16 +10,16 @@ permalink: /changelog/
 [Milestone](https://github.com/mozilla-mobile/android-components/milestone/40?closed=1),
 [API reference](https://mozilla-mobile.github.io/android-components/api/0.38.0/index)
 
-* **browser-engine-system**
-  * Added support for js alerts on SystemEngineView.
-
 * Compiled against:
   * Android (SDK: 28, Support Libraries: 28.0.0)
   * Kotlin (Stdlib: 1.3.10, Coroutines: 1.0.1)
-  * GeckoView (Nightly: 66.0.20181217093726, Beta: 65.0.20181211223337, Release: 64.0.20181214004633)
+  * GeckoView (Nightly: 66.0.20190107093040, Beta: 65.0.20181211223337, Release: 64.0.20181214004633)
   * Mozilla App Services (FxA: 0.12.1, Sync Logins: 0.12.1, Places: 0.12.1)
   * Third Party Libs (Sentry: 1.7.14, Okhttp: 3.12.0)
 
+* **browser-engine-system**
+  * Added support for js alerts on SystemEngineView.
```

Comment:
nit: JavaScript. No need to save characters 🙃

Reply:
👍